Desktop/vsgintersection/Builder.cpp

```diff
@@ -6,7 +6,8 @@ void Builder::setup(vsg::ref_ptr<vsg::Window> window, vsg::ViewportState* viewpo
     auto device = window->getOrCreateDevice();
 
     _compile = vsg::CompileTraversal::create(window);
-    _compile->context.graphicsPipelineStates.emplace_back(viewport);
+    _compile->context.defaultPipelineStates.emplace_back(viewport);
+    _compile->context.overridePipelineStates.emplace_back(vsg::MultisampleState::create(window->getFrameRender().sampleBits));
```

Comment:
If the compilation requires the MultiSampleState then the CompileTraversl constructor should do this, not push this into client application code, that might not know that the window requires multi-sampling.

I'll merge as is, but this issue needs resolved, we can't have frame buffer configuration options spilling out into client code as it'll just cause lots of regression.

FYI, I'm thinking of pass an optional ViewportState pointer into the CompileTraversal constructor along with the Window so that client code needn't worry about the internals of CompileTraversal.

Reply:
I have made the change to the CompileTraversal constructor so the code Builder.cpp is simplified and insulated from the internal implementation details of Context:

https://github.com/vsg-dev/vsgExamples/commit/d10c0cf5eb60eb7ca5bb83195469dd2371b33f7a

This change to vsgExamples multisample branch requires an update to VSG multisample branch to get this change to the CompileTraversal constructor.
https://github.com/vsg-dev/VulkanSceneGraph/commit/67216b561c25a0a2cbdf3100bfc8b6c66ba3d31c